### ISSUE TITLE
impact of changing the semantics of "the resource"

### ISSUE THREAD

**Issue by ajtag** [antleaf/rioxx#46]
Between V2 and V3RC1 I notice the use of the term the resource in the documentation has been split up into concepts that more closely mirrors other catalogging systems concepts  e.g.:

> Work: “a resource reflecting a conceptual essence of the cataloged resource”
> Instance: “a material embodiment of a work”
> Item: “an actual copy (physical or electronic) of an instance”

As I understand dc:identifier now maps onto something like the work and dc:relation to the instance + link to the item.

In the tag definitions, there remains a lot of references to "the resource", without reflecting that the meaning of "the resource" is different from V2 of the format.

Looking through the docs, whilst considering the conceptual split documented above, some mentionions of "the resource" refer to the work, whereas I think the following may be more appropriatly mapped to an instance (This list is open to debate but hopefully illustrates the problem).

- ali:licence_ref
- dc:format
- rioxxterms:type
- rioxxterms:version

e.g. I think it makes sense for dc:format to be part of dc:relation, as if it remains linked to describing the content at dc:identifier it will effectively always be text/html for the mimetype of the repository landing page.

I think this issue existed in V2 and we worked around it, but the move of dc:identifier to be linked to the concept of the work precipitates the meaning of the elements refering back to the resource.

I include the text describing the resource and dc:identifier from the V2 and V3RC1 for reference

<hr />

### V2
The resource refers to the electronic copy of a publication held in a repository, and is that which is being described by the RIOXX metadata record.

This [dc:identifier] field MUST contain an HTTP URI which is a persistent identifier for the resource. The purpose of this field is, through direct identification of the resource, to allow access to it, therefore it is RECOMMENDED that this identifier should point to the actual resource being described by the RIOXX record (typically a file in MS Word or PDF format), rather than to an intermediary resource such as a repository web page. Note that RIOXX does not require any require particular file format to be used for the resource.

To describe another version of the resource with a different identifier, a completely separate RIOXX record should be created.

<hr />

### V3RC1
the resource refers to the resource in the repository which identified by the dc:identifier property in the Rioxx metadata record.

dc:identifier MUST contain an HTTP(S) URI which is a persistent identifier for the resource. In repositories, this is typically a webpage which includes links to other related resources. It is RECOMMENDED that that a DOI, Handle, URN, or other persistent identification scheme be used. In the common case of a "splash-page" linking to related files (potentially in different formats), then one or more instances of the dc:relation property may be included in the Rioxx record to convey this and thereby direct harvesting software agents.

Note that dc:identifier should not be confused with rioxxterms:version_of_record.

**Comment by geo-mac** [comment]
Thanks for querying the semantics, and you are correct that there has been some effort to move towards better modelling of resource concepts. 

We have a number of schema updates to discuss at the next RGG meeting. This will be one. Watch this space!

**Comment by geo-mac** [comment]
Hi @ajtag -- You will note from the [recent RGG minutes](https://www.rioxx.net/governance/minutes/2022-11-07/) that we are considering this issue and will report back in due course. It is actually something we considered in the past but the fact that you have raised it externally gives us reason to revisit it in more detail. Thanks again!

**Comment by ajtag** [comment]
@geo-mac Thanks for picking this up.

After reading the minutes I feel like I should express, or at least clarify, a preference. I tried not to above, but it sounded like I had one represented in the meeting that really isnt my preference.


I very much appreciate moving the new schema design, and see the direction as an improvement on V2.0.
It better represents the the content we have and enables more accurate communication about the resources. 

I raised this case, as I felt with the v2 spec, the metadata fields were not always interpreted as intended, or there was no appropriate field to publish information we want to make available.

The idea of moving to a FRBR style model solves those issues, but my reading of v3rc1, the change felt superficial, only applying the change to the definition of resource, identifier and relation, whilst assuming the other fields apply to the work, rather than the expression where I think many of them apply. 

I am all for moving to the new model, and rolling forward to the more hirearchical model, rather than back to v2, as it means we are able to communicate the approproate metadata, but I also see additional complexity compared to the simplifications of reality encoded in v2. 

The ability to encode content where metadata is missing or unknown, and have records changing, especially in the early life, is a requirement for this schema and I am not sure I can evaluate that before the schema has settled. 

I am only a small consumer of content, so to understand the position orgs CORE and other bodies utiling the spec would be very interesting to see how this also meets their needs. 

I am happy for the balance to be pushed towards the v3 style and think really should be pushed futher so v3 fully embraces  the new model, rather than sitting with a foot still in the v2 camp.
